applications/script.js

```diff
@@ -73,6 +81,10 @@ function closeApplicationDetails() {
   applicationDetailsModal.classList.add('hidden');
   backDropBlur.style.display = 'none';
   document.body.style.overflow = 'auto';
+
+  if (applicationId) {
+    window.location.href = '/applications';
+  }
```

Comment:
- why is this check required?

Reply:
Because when super user go to this route applications/?id=<applicationId> then a particular application modal open and this check is in close modal function by this if the particular modal closes it redirect me to all application page